src/bsp/src/stm32/src/interloc/src/Decawave.cpp

```diff
@@ -305,32 +336,32 @@ void Decawave::retrieveRxFrame(UWBRxFrame* frame) {
         return;
     }
 
-    deca_selectDevice(m_spiDevice);
-
-    // Frame was not received, parse status reg to find the error type
-    if ((m_callbackData.status & SYS_STATUS_ALL_RX_TO) != 0U) {
-        frame->m_status = UWBRxStatus::TIMEOUT;
-        return;
-    }
-
-    if ((m_callbackData.status & SYS_STATUS_ALL_RX_ERR) != 0U) {
-        frame->m_status = UWBRxStatus::ERROR;
-        return;
-    }
-
-    frame->m_statusReg = m_callbackData.status;
-    frame->m_length = m_callbackData.datalength;
-    frame->m_status = UWBRxStatus::FINISHED;
-
-    // Read the frame into memory without the CRC16 located at the end of the frame
-    dwt_readrxdata(frame->m_rxBuffer.data(), m_callbackData.datalength - UWB_CRC_LENGTH, 0);
-    getRxTimestamp(&frame->m_rxTimestamp);
-
-    dwt_readfromdevice(RX_TTCKO_ID, 4, 1, &(frame->m_sfdAngleRegister));
-    // Read information needed for phase calculation
-    uint16_t firstPathIdx = dwt_read16bitoffsetreg(RX_TIME_ID, RX_TIME_FP_INDEX_OFFSET);
-    // Read one extra byte as readaccdata() returns a dummy byte
-    dwt_readaccdata(frame->m_firstPathAccumulator, 5, firstPathIdx);
+    //    deca_selectDevice(m_spiDevice);
+    //
+    //    // Frame was not received, parse status reg to find the error type
+    //    if ((m_callbackData.status & SYS_STATUS_ALL_RX_TO) != 0U) {
+    //        frame->m_status = UWBRxStatus::TIMEOUT;
+    //        return;
+    //    }
+    //
+    //    if ((m_callbackData.status & SYS_STATUS_ALL_RX_ERR) != 0U) {
+    //        frame->m_status = UWBRxStatus::ERROR;
+    //        return;
+    //    }
+    //
+    //    frame->m_statusReg = m_callbackData.status;
+    //    frame->m_length = m_callbackData.datalength;
+    //    frame->m_status = UWBRxStatus::FINISHED;
+    //
+    //    // Read the frame into memory without the CRC16 located at the end of the frame
+    //    dwt_readrxdata(frame->m_rxBuffer.data(), m_callbackData.datalength - UWB_CRC_LENGTH, 0);
+    //    getRxTimestamp(&frame->m_rxTimestamp);
+    //
+    //    dwt_readfromdevice(RX_TTCKO_ID, 4, 1, &(frame->m_sfdAngleRegister));
+    //    // Read information needed for phase calculation
+    //    uint16_t firstPathIdx = dwt_read16bitoffsetreg(RX_TIME_ID, RX_TIME_FP_INDEX_OFFSET);
+    //    // Read one extra byte as readaccdata() returns a dummy byte
+    //    dwt_readaccdata(frame->m_firstPathAccumulator, 5, firstPathIdx);
```

Comment:
Is this still supposed to be commented out?

Reply:
Yeah, I'll be moving this to another function in the next PR so I'm leaving it here for reference. There's an equivalent version being done in the ISR atm.